index.js

```diff
@@ -3,9 +3,8 @@
 var pkg = require('./package.json');
```

Comment:
`var` shows the age of this plugin. You could safely update these to `const` across the plugin if you wanted